recipes/odm2rest/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0a" %}
+
+package:
+  name: odm2rest
+  version: {{ version }}
+
+source:
+  # not useing the jinja variable for the version b/c the release is not pep440
+  # see https://www.python.org/dev/peps/pep-0440/#id25
+  url: https://github.com/ODM2/ODM2RESTfulWebServices/archive/v0.1.0-alpha.tar.gz
+  sha256: f79d94dd58e7ed0e223049067bfc9e1d98234bca9ddf65bfd9eb62ff14d2a8bf
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - django 1.11
+    - jsonschema
+    - odm2api 0.7.*
+    - pandas
+    - python
+    - sqlalchemy
+    # - djangorestframework 3.6.4
+    # - djangorestframework-csv
+    # - djangorestframework-xml
+    # - djangorestframework-yaml
+    # - ipykernel
+    # - numpy
+    # - psycopg2
+    # - pyodbc
+    # - pyyaml
+    # - strict-rfc3339
```

Comment:
All the commented out dependencies are b/c I could not find a `import` for them in the code. Please double check what is needed and I can fix it here.

Reply:
- djangorestframework stuff is absolutely necessary. See https://github.com/ODM2/ODM2RESTfulWebServices/blob/master/odm2rest/settings.py#L31, the rest django-rest-framework stuff are extensions, not explicitly imported.
- ipykernel: probably not an absolute requirement, used mostly to debug in jupyter notebook
- numpy: probably pulled from pandas
- psycopg2 & pyodbc: These are probably pulled with odm2api
- pyyaml: definitely need this. See https://github.com/ODM2/ODM2RESTfulWebServices/blob/master/odm2rest/utils.py#L7. It says `yaml`, but it's using `pyyaml`
- strict-rfc3339: This is needed in the testing making sure that it meets the datetime standard format. So maybe not really needed in production. Thanks.

I hope that's clear. Let me know if you have any questions.

Reply:
> the rest django-rest-framework stuff are extensions, not explicitly imported

Cool! I always learn something new when reviewing your code!

> ipykernel: probably not an absolute requirement, used mostly to debug in jupyter notebook

Maybe we should move that to a `requirements-dev.txt` then? 

> numpy: probably pulled from pandas

Yep, and not used directly as far as I can see. When specifying `numpy` and `pandas` one thinks that the packages needs to link to `numpy` ABI.

> psycopg2 & pyodbc: These are probably pulled with odm2api

Still, they are not imported here, right?

> strict-rfc3339: This is needed in the testing making sure that it meets the datetime standard format. So maybe not really needed in production.

Should also go to `requirements-dev.txt`, right?

> I hope that's clear. Let me know if you have any questions.

Yep. Thanks! I'm re-adding the the rest django-rest-framework stuff.